portal/src/main/webapp/js/src/oncokb/OncoKBConnector.js

```diff
@@ -157,6 +157,7 @@ var OncoKB = (function(_, $) {
     function Instance(id) {
         this.initialized = false;
         this.dataReady = false;
+        this.civicDataReady = false;
```

Comment:
@jjgao We don't have the `OncoKBConnector` in the new refactored code. 

Does civic annotation depend on the data coming from OncoKB? If not, then I don't think it is a good idea to couple civic with OncoKB this way.